plugins/janus_videoroom.c

```diff
@@ -9226,22 +9242,34 @@ static void *janus_videoroom_handler(void *data) {
 									continue;
 								if(videoroom->vcodec[i] == JANUS_VIDEOCODEC_VP9 && vp9_profile) {
 									/* Check if this VP9 profile is available */
-									if(janus_sdp_get_codec_pt_full(offer, -1, janus_videocodec_name(videoroom->vcodec[i]), vp9_profile) != -1) {
+									int pt = janus_sdp_get_codec_pt_full(offer, -1, janus_videocodec_name(videoroom->vcodec[i]), vp9_profile);
+									if(pt != -1) {
 										/* It is */
 										h264_profile = NULL;
 										ps->vcodec = videoroom->vcodec[i];
 										ps->pt = janus_videocodec_pt(ps->vcodec);
+										const char *video_fmtp = janus_sdp_get_fmtp(offer, m->index, pt);
+										if(video_fmtp) {
+											ps->fmtp = g_strdup(video_fmtp);
+										}
```

Comment:
I don't think this is needed anymore, since we're already tracking the profile. Besides, this will cause leaks if `ps->fmtp` exists already, since you're setting the value to a new allocated property and the old one will be lost.